src/model.js

```diff
@@ -214,7 +214,8 @@ export default function createModelClass (name, options, client, metadata) {
     return encodedValue
   }
   Model.get = (...keys) => {
-    return Model.__client__.request(`${Model.__url__}/${keys.join('/')}`, 'GET')
+    let inputPath = keys[0][0] === '/' ? keys.join('/').substr(1) : `${Model.__url__}/${keys.join('/')}`
```

Comment:
A little issue with your naming, the `inputPath` is not a good name, I prefer `resoursePath`.

And about the ternary operator: Nice and tricky, but less readable.

I prefer:

```javascript
if (firstKey.startsWith('/')) {
  /*
  Some good comments.
  */
  resourcePath = .....
}
```